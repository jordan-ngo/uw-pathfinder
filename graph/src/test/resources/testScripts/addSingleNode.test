# Create a graph A
CreateGraph A

# Add one node
AddNode A one

# Output all nodes in the graph
ListNodes A