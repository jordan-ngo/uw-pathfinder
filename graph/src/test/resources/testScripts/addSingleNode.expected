# Create a graph A
created graph A

# Add one node
added node one to A

# Output all nodes in the graph
A contains: one
